### PR TITLE
Upate build.sh and add test 🦭

### PR DESCRIPTION
**Pull request by darcywong00**
This mirrors recent updates to api.keyman.com and keymanweb.com ~of adding tests (lint, phpunit, and link checks) via Composer.~

* Also updates builder.inc.sh

(description updated)
Per review comments, Composer and link checks removed. 

I left a test step for ci.yml to verify it can generate the kmwversions.json file.

### CODE REVIEW

**Comment by mcdurdin** [comment]
Note: we won't do any CI tests at this point so remove composer, link checker, gha to run tests -- we have no real php to check, nor links.

**Comment by darcywong00** [comment]
> Note: we won't do any CI tests at this point so remove composer, link checker, gha to run tests -- we have no real php to check, nor links.

Updated to remove composer, link checker

The GHA just runs the script to generate the static kmwversions.json file

File: build.sh [hunk 1]
```diff
@@ -6,104 +6,128 @@ set -eu
...
 fi
 
-if builder_start_action stop; then
-  # Stop the Docker container
-  _stop_docker_container
-  builder_finish_action success stop
-fi
 
+# Stop the Docker container
+builder_run_action stop _stop_docker_container
```
Comment (mcdurdin) [inline build.sh:85]: This means this will be run on any action, including `configure`. Is that really what we want?

**Review by mcdurdin — CHANGES_REQUESTED** [review]
Looking good, just one question on stopping container

File: build.sh (hunk 1 above)
Comment (darcywong00) [reply build.sh:85]: I thought `builder_run_action` is just shorthand for a single-command action

Comment (mcdurdin) [reply build.sh:85]: Yes, you are right. My bad. Sorry. I must have been tired...

**Review by mcdurdin — APPROVED** [review]

**Pull request merged**